lib-injection/build/docker/python/dd-lib-python-init-test-django-preinstalled/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.9
+
+ENV PYTHONUNBUFFERED 1
+ENV DJANGO_SETTINGS_MODULE django_app
+WORKDIR /src
+ADD . /src
+
+# Install ddtrace beforehand to ensure it doesn't get overridden.
+RUN pip install django ddtrace==1.12.0
```

Comment:
I see this and I think this weblog should not be tested with profiling tests. I also think we should test two things for this app:
- The traces are being sent to the test agent (ok, done).
- The tracer version has not been overwritten by the autoinjection (we don't have this test).

Reply:
Ok, in this case we should skip only the profiling tests

Reply:
Profiling test skipped and latest workflow run is all green